mathesar_ui/src/pages/database/AddEditSchemaModalForm.svelte

```diff
@@ -0,0 +1,137 @@
+<script lang="ts">
+  import { tick } from 'svelte';
+  import type { ModalController } from '@mathesar-component-library';
+  import {
+    CancelOrProceedButtonPair,
+    ControlledModal,
+    TextInput,
+  } from '@mathesar-component-library';
+  import { toast } from '@mathesar/stores/toast';
+  import TextArea from '@mathesar/component-library/text-area/TextArea.svelte';
+
+  export let controller: ModalController;
+  export let getNameValidationErrors: (name: string) => string[];
+  export let getDescriptionValidationErrors: (description: string) => string[];
```

Comment:
Since we're not validating description, why do we need this additional prop and code around it?

I think we can remove the unnecessary code blocks in this component surrounding description validation.

Reply:
I kept it if later we want to add some validations but I agree that's just more mantainance for the code you MIGHT need later. Hence removed it